strongbox-cron-tasks/src/test/java/org.carlspring.strongbox.cron/RebuildMavenIndexesCronJobTest.java

```diff
@@ -0,0 +1,307 @@
+            Storage newStorage = new Storage("storage1");
+            Repository repository3 = new Repository("releases");
+            repository3.setPolicy(RepositoryPolicyEnum.RELEASE.getPolicy());
+            repository3.setStorage(newStorage);
+            configurationManagementService.addOrUpdateStorage(newStorage);
+            repositoryManagementService.createRepository("storage1", "releases");
+            newStorage.addOrUpdateRepository(repository3);
+
+            //Create released artifact
+            artifact4 = generateArtifact(REPOSITORY_BASEDIR_3.getAbsolutePath(), ga1 + ":1.0:jar");
+
+            changeCreationDate(artifact1);
+            changeCreationDate(artifact2);
+            changeCreationDate(artifact3);
+            changeCreationDate(artifact4);
+
+            initialized = true;
+        }
+    }
+
+    public void addRebuildCronJobConfig(String name,
+                                        String storageId,
+                                        String repositoryId,
+                                        String basePath)
+            throws ClassNotFoundException, CronTaskException, InstantiationException, SchedulerException,
+                   IllegalAccessException
+    {
+        CronTaskConfiguration cronTaskConfiguration = new CronTaskConfiguration();
+        cronTaskConfiguration.setName(name);
+        cronTaskConfiguration.addProperty("jobClass", RebuildMavenIndexesCronJob.class.getName());
+        cronTaskConfiguration.addProperty("cronExpression", "0 0/10 * 1/1 * ? *");
+        cronTaskConfiguration.addProperty("storageId", storageId);
+        cronTaskConfiguration.addProperty("repositoryId", repositoryId);
+        cronTaskConfiguration.addProperty("basePath", basePath);
+
+        cronTaskConfigurationService.saveConfiguration(cronTaskConfiguration);
+        CronTaskConfiguration obj = cronTaskConfigurationService.findOne(name);
+        assertNotNull(obj);
+    }
+
+    public void deleteRebuildCronJobConfig(String name)
+            throws SchedulerException, CronTaskNotFoundException, ClassNotFoundException
+    {
+        List<CronTaskConfiguration> confs = cronTaskConfigurationService.getConfiguration(name);
+
+        confs.forEach(cronTaskConfiguration ->
+                      {
+                          assertNotNull(cronTaskConfiguration);
+                          try
+                          {
+                              cronTaskConfigurationService.deleteConfiguration(cronTaskConfiguration);
+                          }
+                          catch (Exception e)
+                          {
+                              throw new RuntimeException(e);
+                          }
+                      });
+
+        assertNull(cronTaskConfigurationService.findOne(name));
+    }
+
+    @Test
+    public void testRebuildArtifactsIndexes()
+            throws NoSuchAlgorithmException, XmlPullParserException,
```

Comment:
I think in test cases like these, it would be fine to just replace such huge lists of thrown exceptions with just `throws Exception`. Would you mind correcting this across the other test methods?